cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -951,6 +952,8 @@ public Row next() {
       } else {
         order = Bytes.compareTo(currentKey, currentRow.getRow());
       }
+
+      @SuppressWarnings("UnusedAssignment")
```

Comment:
Seems like better to remove the `= null;` assignment instead of adding this annotation.